scout/server/blueprints/variants/templates/variants/variants.html

```diff
@@ -176,7 +176,17 @@
     var chromSel = document.getElementById("chrom");
     chromSel.addEventListener("change", function() {
       populateCytobands({{cytobands|safe}});
+      updateCoordinateFields(event.target)
```

Comment:
I've noticed that while the search field is present in the SNVs and SVs pages, it's missing in the search form of cancer variants and SV cancer variants. It would be very useful if you could add it there too